install_config/adding_hosts_to_existing_cluster.adoc

```diff
@@ -309,3 +309,133 @@ $ ansible-playbook [-i /path/to/file] \
 
 . After the playbook completes successfully,
 xref:../install_config/install/advanced_install.adoc#advanced-verifying-the-installation[verify the installation].
+
+[[replacing-existing-masters]]
+== Replacing Existing Masters
+
+. Back up etcd from the primary
+xref:../admin_guide/backup_restore.adoc#etcd-backup[etcd] and
+xref:../admin_guide/backup_restore.adoc#master-backup[master].
++
+[IMPORTANT]
+====
+Ensure that you back up the *_/etc/etcd/ca/_* directory, as noted in the
+xref:../admin_guide/backup_restore.adoc#backup-restore-adding-etcd-hosts[Backup
+and Restore] topic.
+====
+
+. Provision as many new virtual machines (VMs) as there are masters to replace.
+
+. Add or expand the cluster. If you want to add 3 masters with etcd collated,
+ repeat this section 3 times.
+
+.. Add a master using the xref:adding-nodes-advanced[master *_scaleup.yml_*
```

Comment:
Maybe make parallel with the ectd step? Like:
Add a xref:adding-nodes-advanced[master]. In step 3 of that process, add the host of the
new data center in `[new_masters]` and `[new_nodes] and run the master *_scaleup.yml_* playbook.